suite-native/module-send/src/components/ReviewOutputItemList.tsx

```diff
@@ -1,21 +1,28 @@
 import { useSelector } from 'react-redux';
 
-import { ErrorMessage, Loader, Text } from '@suite-native/atoms';
+import { ErrorMessage, VStack } from '@suite-native/atoms';
 import { AccountKey } from '@suite-common/wallet-types';
 import {
     AccountsRootState,
     DeviceRootState,
     SendRootState,
     selectAccountByKey,
-    selectSendPrecomposedTx,
+    selectSendSignedTx,
 } from '@suite-common/wallet-core';
-import { CryptoAmountFormatter } from '@suite-native/formatters';
 
 import { ReviewOutputItem } from './ReviewOutputItem';
-import { selectTransactionReviewOutputs } from '../selectors';
+import {
+    selectTransactionReviewActiveStepIndex,
+    selectTransactionReviewOutputs,
+} from '../selectors';
+import { ReviewOutputSummaryItem } from './ReviewOutputSummaryItem';
+import { SlidingFooterOverlay } from './SlidingFooterOverlay';
 
 type ReviewOutputItemListProps = { accountKey: AccountKey };
 
+const INITIAL_OFFSET = 230;
+const STEP_HEIGHT = 160;
```

Comment:
I wanted to fix this later when there will be other output types supperted (having different heights), but you are right that now a change of fontSize would break it. 

I make the component dynamic, so it reads heigh of it childs via the `onLayout` event. 8867fec3ab